### DISCUSSION TITLE
My original account got flagged because of reporting malicious activities and there's no response after almost 2 months

### DISCUSSION THREAD

**Discussion by AliMilani13** [community/community#142087] [New to GitHub]
Hello everyone. 

my original account  [Ali-milani-13](https://github.com/Ali-Milani-13) was flagged as a spam for reporting a malware distribution in the discussion section of the [https://github.com/hiddify/hiddify-next](https://github.com/hiddify/hiddify-next). 

someone sent me an encrypted binary, telling me, it is a fix and then I figured out it was a Trojan file and reported that person. after that, he removed his account and I got flagged as spam. 

I've submitted support ticket from my account but there's no response what so ever after 2 months. 

I even filed a support ticket from this account and they didn't answer it either. 

I don't know why they are ghosting their users who use this website at work. 

and this was the discussion I was talking about: 

[https://github.com/hiddify/hiddify-next/issues/1281](https://github.com/hiddify/hiddify-next/issues/1281)
![image](https://github.com/user-attachments/assets/1808bbde-5bf3-432b-b4ab-1ff00df378c6)

**Comment by ghostinhershell** [comment]
Hi @AliMilani13,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by AliMilani13** [reply]
> Hi @AliMilani13,
> 
> Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.
> 
> As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
> 
> Thank you

Thank you so much. I would be glad to wait another year and suspend my whole job to get my answer about unjust flagging without no warrant. of course resources are limited. couple billions of dollars from Microsoft is nothing. 😊
